Projects/TerminateContracts/Sources/Service/OctopusImplementation/TerminateContractsServiceOctopus.swift

```diff
@@ -0,0 +1,36 @@
+import Flow
+import Presentation
+import hGraphQL
+
+public class TerminateContractsOctopus: TerminateContractsService {
+    public init() {}
+
+    public func startTermination(contractId: String) -> FiniteSignal<TerminationContractAction> {
+        let mutation = OctopusGraphQL.FlowTerminationStartMutation(
+            input: OctopusGraphQL.FlowTerminationStartInput(contractId: contractId),
+            context: nil
+        )
+        return mutation.execute(\.flowTerminationStart.fragments.flowTerminationFragment.currentStep)
```

Comment:
@sladan-hedvig not sure where we want to execute this

Reply:
Did sine updates to this 👍 now it should be fine